solutions/swb-app/src/environments/environment.ts

```diff
@@ -0,0 +1,38 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { EnvironmentStatus } from '@aws/workbench-core-environments';
+
+export interface Environment {
+  id: string;
+  instanceId: string | undefined;
+  cidr: string;
+  description: string;
+  error: { type: string; value: string } | undefined;
+  name: string;
+  outputs: { id: string; value: string; description: string }[];
+  projectId: string;
+  status: EnvironmentStatus;
+  provisionedProductId: string;
+  envTypeConfigId: string;
+  updatedAt: string;
+  updatedBy: string;
+  createdAt: string;
+  createdBy: string;
+  owner: string;
+  type: string;
+  dependency: string;
+  //eslint-disable-next-line @typescript-eslint/no-explicit-any
+  ETC?: any;
+  //eslint-disable-next-line @typescript-eslint/no-explicit-any
+  PROJ?: any;
+  // TODO: Replace any[] with <type>[]
+  //eslint-disable-next-line @typescript-eslint/no-explicit-any
+  DATASETS?: any[];
```

Comment:
I agree on that, for now I only translated directly what we had in the json schema and made it reflect the api specifications, I am planning on creating an item to address this since we are showing `pk` and `sk` on the response for those dependencies and the fix would extended the scope of this task even more

Reply:
task GALI-2436 created to address this